yt_dlp/extractor/youtube.py

```diff
@@ -2485,6 +2402,7 @@ def _extract_player_responses(self, clients, video_id, webpage, master_ytcfg, pl
         while clients:
             client = clients.pop()
             player_ytcfg = master_ytcfg if client == 'web' else {}
+            r'''  # This doesnt have thirdParty set - Remove?
```

Comment:
I say remove. hard coded configs are fairly safe and getting the latest one for a client that is rarely used is unnecessary.

Reply:
actually it doesn't hurt keeping the config download, but the age-gate check here can go.